app/models/environment.rb

```diff
@@ -25,6 +26,15 @@ def active_lock
     locks.active.first
   end
 
+  # Configures this environment to auto deploy the given branch.
+  def configure_auto_deploy(branch, options = {})
+    self.update_attributes!(auto_deploy_branch: branch, auto_deploy_user: options[:user])
```

Comment:
Yep. Basically, whoever configures auto deploy on the repo will be the user that SlashDeploy uses the create the github deployment. 

Would be nice to have the option to just pass it a github access token and use that instead.